src/kubernetes/server/server-deployment.yaml

```diff
@@ -35,6 +35,8 @@ spec:
           value: 'false'
         - name: SPRING_CLOUD_DATAFLOW_FEATURES_ANALYTICS_ENABLED
           value: 'true'
+        - name: SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
```

Comment:
I think we should have this enabled by default.   But that differs from CF and the upcoming local.  @sabbyanandan can we make an exception here?